pages/mesosphere/dcos/services/elastic/3.2.0-7.9.3/release-notes/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Discover the new features, updates, and known limitations in this release of the Elastic Service
+title: Release Notes
+menuWeight: 10
+model: /mesosphere/dcos/services/elastic/data.yml
+render: mustache
+---
+
```

Comment:
@alexeygorobets we need to put a release date in here before we can merge.

Reply:
added release date